### DISCUSSION TITLE
Problema ejecutando main.c

### DISCUSSION THREAD

**Discussion by GitCross135** [IIC2133-PUC/2024-1#49] [Tarea 0] [answered]
Hola, estoy teniendo problemas al ejecutar el archivo main.c desde la terminal 😭 Me dice Permission denied. ayuda :(

![image](https://github.com/IIC2133-PUC/2024-1/assets/130419134/cab19f51-67fc-41aa-92f7-ea4aecd9ed5e)

**Comment by PabloNavarroY** [comment, marked as answer]
Tienes que **compilar** tu programa para poder tener un **ejecutable**, por eso tenías que instalar gcc, un **compilador**. Para hacer eso solo tienes que ocupar el comando `make`.
